lib/LaTeXML/Post/MathML.pm

```diff
@@ -715,13 +715,15 @@ sub stylizeContent {
   my $u_variant  = $variant
     && ($plane1hack ? $plane1hackable{$variant}
     : ($plane1 ? $variant : undef));
-  my $u_text = $u_variant && unicode_convert($text, $u_variant);
+  my $u_text = ($tag ne 'm:mtext') && $u_variant && unicode_convert($text, $u_variant);
   if ((defined $u_text) && ($u_text ne '')) {    # didn't remap the text ? Keep text & variant
     $text    = $u_text;
     $variant = ($plane1hack && ($variant ne $u_variant) && ($variant =~ /^bold/)
       ? 'bold' : undef); }                       # Possibly keep variant bold
                                                  # Use class (css) to patchup some weak translations
-  if    (!$font) { }
+  if ($text =~ /^\p{Format}*$/) {                # Formatting (eg. InvisibleTImes)
```

Comment:
this regex is a little odd - does it mean starting with `\p{Format}` or empty? If so, you can replace the `*$` with `?`

Reply:
I clarified the comment; should mean that the text contains only formatting characters, or is empty.